page/layouts/partials/top-navigation.html

```diff
@@ -0,0 +1,9 @@
+<div id="contao-network">
+  <a href="https://contao.org" title="contao.org" class="contao active"><i></i> <span>Contao</span></a>
+  <a href="https://docs.contao.org" title="docs.contao.org" class="docs"><i></i> <span>Docs</span></a>
+  <a href="https://community.contao.org" title="community.contao.org" class="community"><i></i> <span>Community</span></a>
+  <a href="https://themes.contao.org" title="themes.contao.org" class="themes"><i></i> <span>Themes</span></a>
+  <a href="https://extensions.contao.org" title="extensions.contao.org" class="extensions"><i></i> <span>Extensions</span></a>
+  <a href="https://fanshop.contao.org" title="fanshop.contao.org" class="fanshop"><i></i> <span>Fanshop</span></a>
+  <a href="https://login.contao.org" title="login.contao.org" class="login"><i></i> <span>Login</span></a>
+</div>
```

Comment:
We integrate the network HTML everywhere via the https://contao.org/files/js/network.min.js

Reply:
Done.